README.md

```diff
@@ -64,6 +70,7 @@ Describe the rules of scope in JavaScript.
 
 Your Answer:
 ```text
+Scope is the container created to house different areas of your code to ensure the JavaScript behaves the way you want it to.
```

Comment:
I like this definition.  We would like to hear a little bit more.  Maybe some discussion of when scopes are created.  How to control variable scope.